dspace-spring-rest/src/test/java/org/dspace/app/rest/builder/RequestItemHelperDAO.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.rest.builder;
+
+import java.sql.SQLException;
+import javax.persistence.Query;
+
+import org.dspace.app.requestitem.RequestItem;
+import org.dspace.core.AbstractHibernateDAO;
+import org.dspace.core.Context;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Supply missing "delete" operation on RequestItem, to support testing.
```

Comment:
why not create it right now in the RequestItemService?

Reply:
It seemed so strange to me that delete was not implemented, that it might have been a conscious decision to make these requests un-deletable.  This needs discussion.  I have no objection to implementing deletion, but I don't want to undo a design choice through ignorance.

Reply:
Delete operation moved to main code.